### ISSUE TITLE
Hermite polynomials

### ISSUE THREAD

**Issue by LeilaSalomon** [mfem/mfem#2617]
Hello,

Does it exist a finite-element space that ensures the continuity of the solution and its derivative? Like the Hermite polynomials ?

Best,

Leïla

**Comment by tzanio** [comment]
Yes, there are C1 elements that are in H2, i.e. have both continuous values and continuous derivatives.

These however are not currently supported in MFEM.
